src/btree/bt_sync_obsolete.c

```diff
@@ -11,6 +11,31 @@
 #define WT_CHECKPOINT_CLEANUP_FILE_INTERVAL 1 /* 1 second */
 #define WT_URI_FILE_PREFIX "file:"
 
+/*
+ * __sync_obsolete_limit_reached --
+ *     This function checks whether checkpoint cleanup can continue given the work done so far and
```

Comment:
How about adding `obsolete time window pages` in the comment as follows:
This function checks whether checkpoint cleanup can continue `operating on the obsolete time window pages`?

Reply:
Updated in e7f0aa1aded1ea7af5414a3bbd7748ee7083e845